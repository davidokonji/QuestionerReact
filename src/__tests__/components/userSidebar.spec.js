import React from 'react';
import { shallow } from 'enzyme';
import UserSidebar from '../../components/userSidebar';

let wrapper;

describe('User Sidebar', () => {
  beforeAll(() => {
    wrapper = shallow(<UserSidebar />);
  });
  it('should render component correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render component with props', () => {
    const props = {
      images: [''],
      firstname: '',
      lastname: '',
      username: '',
      about: ''
    };
    wrapper = shallow(<UserSidebar {...props} />);
  });
});
