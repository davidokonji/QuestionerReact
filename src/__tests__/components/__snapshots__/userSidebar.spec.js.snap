// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Sidebar should render component correctly 1`] = `
<div
  className="col-12 col-md-3 shadow-sm ml-md-5 ml-4 mt-3"
>
  <div
    className="m-5"
  >
    <div
      className="col-12"
    >
      <img
        alt="profile"
        className="rounded-circle my-3"
        style={
          Object {
            "height": "18rem",
            "objectFit": "cover",
            "width": "18rem",
          }
        }
      />
    </div>
    <div
      className="my-3"
    >
      <h2>
         
      </h2>
    </div>
    <div
      className="my-3"
    />
    <div
      className="my-3"
    />
    <div
      className="my-3"
    >
      <Button
        styles="btn p-0"
        text="edit profile"
      />
    </div>
  </div>
</div>
`;
